### PR TITLE
Cico tests

### PR DESCRIPTION
**Pull request by Raunak1203**

### CODE REVIEW

File: EE_API_automation/cico_run_EE_tests.sh [hunk 1]
```diff
@@ -14,7 +14,8 @@ if [ -e "../jenkins-env" ]; then
 fi
 
 echo installing dependency
-sudo yum -y install python2 python2-pytest python2-requests python2-jmespath
+sudo yum -y install python2 python-pip
```
Comment (rgarg1) [inline EE_API_automation/cico_run_EE_tests.sh]: Do we need to enable EPEL repository before installing pip? sudo yum install epel-release

Comment (rgarg1) [inline EE_API_automation/cico_run_EE_tests.sh]: sudo yum -y install python2 python-pip OR
sudo yum -y install python python-pip

Comment (Raunak1203) [reply EE_API_automation/cico_run_EE_tests.sh]: Yes, we need to enable epel...added it

Comment (rgarg1) [reply EE_API_automation/cico_run_EE_tests.sh]: The automation is based on Python2.

**Pull request merged**